packages/cli/cli/src/cli.ts

```diff
@@ -328,6 +330,46 @@ function addIrCommand(cli: Argv<GlobalCliOptions>, cliContext: CliContext) {
     );
 }
 
+function addFdrCommand(cli: Argv<GlobalCliOptions>, cliContext: CliContext) {
+    cli.command(
+        "fdr <path-to-output>",
+        false, // hide from help message
+        (yargs) =>
+            yargs
+                .positional("path-to-output", {
+                    type: "string",
+                    description: "Path to write FDR API definition",
+                    demandOption: true,
+                })
+                .option("api", {
+                    string: true,
+                    description: "Only run the command on the provided API",
+                })
+                .option("language", {
+                    choices: Object.values(GenerationLanguage),
+                    description: "Generate FDR API definition for a particular language",
+                })
```

Comment:
The `language` should not matter for the FDR API Definition, so we should be able to remove it. 

This matters for IR because the IR generated for each language is different bc the IR handles escaping reserved keywords for each language, which is configured [here](https://github.com/fern-api/fern/blob/main/packages/cli/generation/ir-generator/src/casings/reserved.ts)